File: src/main/java/axirassa/webapp/pages/trigger/CreateTrigger.java

package axirassa.webapp.pages.trigger;

import org.apache.shiro.authz.annotation.RequiresUser;
import org.apache.tapestry5.annotations.Property;

@RequiresUser
public class CreateTrigger {
	@Property
	private String latency;

	@Property
	private String responseTime;

	@Property
	private String responseSize;
}

File: src/main/java/axirassa/webapp/pages/trigger/CreateTrigger.scala
package axirassa.webapp.pages.trigger

import org.apache.tapestry5.annotations.Property
import org.apache.shiro.authz.annotation.RequiresUser

@RequiresUser
class CreateTrigger {
  @Property
  var latency : String = _

  @Property
  var responseTime : String = _

  @Property
  var responseSize : String = _
}